build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1690,4 +1690,21 @@ message RequestMetadata {
   // An identifier to tie multiple tool invocations together. For example,
   // runs of foo_test, bar_test and baz_test on a post-submit of a given patch.
   string correlated_invocations_id = 4;
+
+  // A one-word description of the kind of action, for example, CppCompile or GoLink.
+  // There is no standard agreed set of values for this, and they are expected to vary between different client tools.
+  string action_mnemonic = 5;
+
+  // A namespace within which the build was performed, for instance a repository/workspace name.
```

Comment:
Consider also adding something about the expected (or not) global uniqueness of this. E.g. "May not be globally unique, as with multiple users having workspaces of the same name".